data/pigui/modules/hyperjump-planner.lua

```diff
@@ -139,7 +139,8 @@ end --mainButton
 
 local function buildJumpRouteList()
 	hyperjump_route = {}
-	local start = current_path
+	local player = Game.player
```

Comment:
Are you sure it should be `local` ? looks like player is a global to me. It's already defined on L14, and assigned in L326

Reply:
@impaktor since this is a pointer, there is no difference inside this function. But the definition on line 326 only works when the hj-planner window is visible. It may happen that this function `buildJumpRouteList()` will be called _before_ the hj-planneer is shown. and global `player` will be `nil`.  In the other hand, other functions should not rely on the assignment in `buildJumpRouteList()`. Therefore, I decided not to touch the global variable `player` in this function.